File: solutions/ruby/raindrops/1/raindrops.rb
module Raindrops
  def Raindrops.convert(number)
    result = pling(number) + plang(number) + plong(number)
    result.empty? ? number.to_s : result
  end

  private
  def self.pling(number)
    number % 3 == 0 ? 'Pling' : ''
  end

  def self.plang(number)
    number % 5 == 0 ? 'Plang' : ''
  end

  def self.plong(number)
    number % 7 == 0 ? 'Plong' : ''
  end
end

File: solutions/go/raindrops/1/raindrops.go
package raindrops

import "strconv"

func Convert(value int) string {
	output := ""
	if value % 3 == 0{
		output += "Pling"
	}
	if value % 5 == 0{
		output += "Plang"
	}
	if value % 7 == 0{
		output += "Plong"
	}
	if output == "" {
		return strconv.Itoa(value)
	}
	return output
}